example/set-welcome-message.html

```
@@ -0,0 +1,39 @@
+  <h1> This sample code will help you to set the welcome message for different web pages</h1>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)